audit_exceptions/provided_by_macos_depends_on_allowlist.json

```diff
@@ -1,6 +1,7 @@
 [
   "apr",
   "apr-util",
+  "gnu-getopt",
```

Comment:
Please remove this. We can allow the dependency instead.

Reply:
This was discussed in https://github.com/Homebrew/homebrew-core/issues/165752
@Bo98 Can you comment on this? By the nature of the `gnu-getopt` package, all new packages will fail at `brew audit --online --new`, because the dependent package are **supposed** to use `uses_from_macos`. For any dependents of `gnu-getopt` this seems like an unnecessary barrier for new packages.

What I proposed in the issue is to cherry-pick this commit, but I haven't heard any feedback on if I should go ahead and make a PR about it since then.

Reply:
> For any dependents of gnu-getopt this seems like an unnecessary barrier for new packages.

It's not, it just requires a maintainer to override the check before merging. This exists as a "human please check this".